ShinyExpenseTrackerTests/src/ca/ualberta/cs/shinyexpensetracker/test/TestExpenseItemDetailView.java

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
As mentioned before, Javadoc class comments should appear right above the class definition.